optaplanner-examples/src/main/java/org/optaplanner/examples/cloudbalancing/optional/domain/CloudComputerStrengthComparator.java

```diff
@@ -19,18 +19,21 @@
 import java.io.Serializable;
 import java.util.Comparator;
 
-import org.apache.commons.lang3.builder.CompareToBuilder;
 import org.optaplanner.examples.cloudbalancing.domain.CloudComputer;
 
 public class CloudComputerStrengthComparator implements Comparator<CloudComputer>, Serializable {
 
+    private static final Comparator<CloudComputer> DESCENDING_COST_COMPARATOR =
+            Comparator.comparing(CloudComputer::getCost).reversed();
+
+    private static final Comparator<CloudComputer> COMPARATOR =
+            Comparator.comparingInt(CloudComputer::getMultiplicand)
+            .thenComparing(DESCENDING_COST_COMPARATOR) // Descending (but this is debatable)
```

Comment:
I'd inline the DESCENDING_COST_COMPARATOR here, it's not like we need it anywhere else

Reply:
See also https://stackoverflow.com/a/30382620/472109